apps/coordinator/src/components/ScriptExplorer/SignatureImporter.jsx

```diff
@@ -579,7 +593,8 @@ SignatureImporter.propTypes = {
     }),
   ).isRequired,
   fee: PropTypes.string.isRequired,
-  inputs: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  inputs: PropTypes.arrayOf(PropTypes.shape({ amountSats: PropTypes.string }))
```

Comment:
it actually has mre than this but this value is directly used and so an error was being thrown for not having it declared. I didn't spend too much time building out the proptypes as typescript will obsolete this.